sourcetools/objectmodel/com/ibm/j9tools/om/io/BuildInfoParser.java

```diff
@@ -103,11 +103,13 @@ public void startElement(String uri, String localName, String qName, Attributes
 			/* Remember where this build spec was defined */
 			_buildInfo.setLocation(_documentLocatorFile, _documentLocator);
 
-			String schemaSource = _parser.getProperty(AbstractParser.JAXP_SCHEMA_SOURCE).toString();
-			int schemaVersion = Integer.parseInt(schemaSource.substring(schemaSource.length() - 5, schemaSource.length() - 4));
-
-			if (schemaVersion >= 4) {
-				_buildInfo.setValidateDefaultSizes(true);
+			Object schemaSourceProp = _parser.getProperty(AbstractParser.JAXP_SCHEMA_SOURCE);
+			if (schemaSourceProp != null) {
```

Comment:
I think `(schemaSourceProp == null)` signals something is wrong: we shouldn't be ignoring that.

Reply:
Will take a look at this but could this be done once the code is pushed as we need to get this PR pushed into the branch so other people can use it rather than my personal fork/branch.

Reply:
We shouldn't be pushing incorrect code to meet an agenda. Is this change still needed? Why is the property null on z/OS?

Reply:
I will remove the change and see if the error still occurs. 
If it does I will investigate why it is occurring on z/OS.

Reply:
The latest code builds correctly without these earlier changes so I have removed them.